src/screens/ImportData.js

```diff
@@ -394,6 +394,7 @@ const ImportFormContainer = styled(Column)`
   height: 100%;
   background-color: #4F4F4F;
   align-items: center;
+  overflow: scroll; 
```

Comment:
Actually, this should be `overflow-y: auto;`, otherwise scrollbars will appear when unnecessary.